src/Microsoft.Health.Dicom.Api/Controllers/CustomTagController.cs

```diff
@@ -50,5 +51,51 @@ public async Task<IActionResult> PostAsync([FromBody] IEnumerable<CustomTagEntry
             return StatusCode(
                (int)HttpStatusCode.Accepted, response);
         }
+
+        /// <summary>
+        /// Handles requests to get all custom tags.
+        /// </summary>
+        /// <returns>
+        /// Returns Bad Request if given path can't be parsed. Returns Not Found if given path doesn't map to a stored
+        /// custom tag or if no custom tags are stored. Returns OK with a JSON body of all tags in other cases.
+        /// </returns>
+        [ProducesResponseType(typeof(JsonResult), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [HttpGet]
+        [Route(KnownRoutes.CustomTagRoute)]
+        [AuditEventType(AuditEventSubType.GetAllCustomTags)]
+        public async Task<IActionResult> GetAllTagsAsync()
```

Comment:
I'm thinking if we could do similar thing like GetFramesAsync (https://github.com/microsoft/dicom-server/blob/master/src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs) which can support return multiple either multiple or single frame

Reply:
Per our conversation during standup, I'll maintain it like this to follow REST API guidelines to not have trailing whitespaces in uris.